ReleaseProcedure.txt

```diff
@@ -25,9 +25,9 @@ Tools\gitreleasemanager\0.7.0\tools\GitReleaseManager.exe create -o nhibernate -
     If the release tag does not match the released version with major.minor.release
     formalism, the NuGet package will have an invalid link to release notes.
 
-  * Check/update version number in common.xml and NHibernate.props under
-    build-common folder, in master.xml under doc\reference folder, and in
-    appveyor.yml in the NHibernate root.
+  * Check/update version number in NHibernate.props under build-common folder.
+    Clear VersionSuffix if present.
+    NOTE: This paragraph is changed for 5.3 - if you are releasing earlier version please check instruction from appropriate branch   
 
   * Don't forget to commit the above.
 
```

Comment:
I think a final step should be added to finish the release : preparing the master branch for next version builds. And by the way I propose another missing step:

```
  * If this was the master branch, create the stable branch for the new release, named
    according to its version with "x" instead of its patch number. By example, "5.3.x".
  * Increment the version number minor in NHibernate.props under build-common folder
    and set VersionSuffix to "dev", and commit it directly (without going through a PR).
```

Reply:
Added with some adjustments.  But I don't think that's last step is mandatory and should be always done during release (as next version might be yet undecided). Feel free to adjust it further directly.